tests/system/src/test/java/test/com/sun/marlin/DashedRectTest.java

```diff
@@ -97,6 +95,46 @@ public void init() {
         public void start(Stage primaryStage) throws Exception {
             this.stage = primaryStage;
 
+            stage.setTitle("DashedRectTest");
```

Comment:
I recommend setting a dummy Scene before showing the Stage, otherwise it can render garbage (at least on Linux it does), which could lead to someone thinking there is something wrong:

```
        stage.setScene(new Scene(new Group()));
```

Reply:
fixed